js/scripts.js

```diff
@@ -0,0 +1,46 @@
+// Jacob here, found something which might work.
+// In the html, give the section/header/div the following id to show the clock:: 'clock'
+// For example: <h1 class="text-center" id="clock"></h1>
+
+// MAKE THE ARRAYS HERE.
+// (will need at least 2: one for the titles and one for the actual description)
+
+// PUT THE boolean VALUE HERE (for preventing the users from spamming the "refresh" feature)
+var hammertime = true;
```

Comment:
Unless absolutely necessary, let's use the "let" and "const" for ES6 unless it's absolutely necessary to use "var" somewhere.

Try to keep the variable names clear so someone new to the code can understand it without having to read comment explanations, if possible. Something like refreshCheck, refreshSpamBlocker, or something if you feel would be more clear.